app/models/good_job/lockable.rb

```diff
@@ -37,11 +37,14 @@ module Lockable
       # @param function [String, Symbol]  Postgres Advisory Lock function name to use
       # @return [ActiveRecord::Relation]
       #   A relation selecting only the records that were locked.
-      scope :advisory_lock, (lambda do |column: _advisory_lockable_column, function: advisory_lockable_function|
+      scope :advisory_lock, (lambda do |column: _advisory_lockable_column, function: advisory_lockable_function, queue_select_limit: nil|
```

Comment:
Within `Lockable` I think the option should be named `select_limit` (rather than `queue_`) because I'm _intending_ `Lockable` to be extractable and usage/purpose neutral.

btw, nice naming of `queue_select_limit` generally. I was sweating the naming of this option 👍🏻

Reply:
Thank you! I renamed the variable/argument in that file.